### ISSUE TITLE
Keys are lowercased in facet distribution

### ISSUE THREAD

**Issue by MarinPostma** [meilisearch/MeiliSearch#786]
When asking for a facet distribution, the keys are returned lowercased. They should be returned with the case specified by the user.

**Comment by curquiza** [comment]
Hello! Still have the issue (if it's indeed the same issue)

- create an index
- add:
```json
{ "id": 2,    "title": "Pride and Prejudice", "author": "Jane Austin",  "genre": "Romance" },
```
- update the settings to set `genre` as `attributesForFaceting`
- get the documents:
```json
[
    {
        "id": 2,
        "title": "Pride and Prejudice",
        "author": "Jane Austin",
        "genre": "Romance"
    }
]
```
-> OK
- search:
```json
{
    "q": "pride",
    "facetsDistribution": ["genre"]
}
```
```json
{
    "hits": [
        {
            "id": 2,
            "title": "Pride and Prejudice",
            "author": "Jane Austin",
            "genre": "Romance"
        }
    ],
    "offset": 0,
    "limit": 20,
    "nbHits": 1,
    "exhaustiveNbHits": false,
    "processingTimeMs": 2,
    "query": "pride",
    "facetsDistribution": {
        "genre": {
            "romance": 1
        }
    },
    "exhaustiveFacetsCount": true
}
```

**Comment by MarinPostma** [comment]
There was an issue when this pr was merged, #901 fixes that.
